src/auth/passwordreset/components/ResetComponent.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ResetPassword from './ResetForm';
+import resetUtil from '../redux/validations';
+import withPasswordState from '../containers/withPasswordState';
+
+export class ResetPasswordContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      errors: {},
+      success:{}
+    };
+  }
+
+  onChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value , errors:{}, success:{}});
+  }
+
+  checkValidEmail = (userEmail) => {
+    const mailFormat = resetUtil.RESET_UTILS;
+    if (userEmail.match(mailFormat)) {
+      return true;
+    }
+    return false;
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const {email} = this.state
+    const userdata = {
+      email: email
+    };
+    
+    if (email === "" ) {
+      this.setState({ errors: { userEmail: 'Email field required' } })
+    } else if (!this.checkValidEmail(email)) {
+      this.setState({ errors: { userEmail: 'Invalid email format' } });
+    }
+    else {
+      this.props.resetPasswordAction(userdata, this.props);
```

Comment:
Don't pass in the whole props object, pass in only what you need

Reply:
This has been resolved, thank you.